build.gradle

```diff
@@ -157,7 +157,8 @@ task generateApiOutputFormat(type: cz.habarta.typescript.generator.gradle.Genera
 
 task generateApiRequestFormat(type: cz.habarta.typescript.generator.gradle.GenerateTask) {
     classPatterns = [
-        "teammates.ui.webapi.request.**"
+        "teammates.ui.webapi.request.**",
+        "teammates.ui.webapi.action.Intent"
```

Comment:
Why not just move `Intent.java` into the request folder?

Reply:
@xpdavid Moved `Intent.java` from action to request folder.